### PR TITLE
Implementacion del card Home page: Frontend - News

### PR DESCRIPTION
**Pull request by FloresFelix**
Summary
implementación del card Home page: Frontend - News

Screenshot

Trello card
https://trello.com/c/xXBJJpOl

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt]: Never forget run the "Code -> Reformat Code" Android Studio tool to make sure that your code is well indented.

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: Rename this presenter function to onSwipeRefresh(), remember that the fragment notify to presenter asking what should I do? Usually, the fragment don't take decision by itself.

File: app/src/main/java/ar/com/wolox/android/example/model/ItemNewsModel.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/model/ItemNewsModel.kt:9]: You will need to change this names when you perform the integration with backend.

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt]: `holder.apply {} `

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsAdapter.kt]: move the date format to a constant

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: `with(binding)`

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: Use "with" function like I mentioned above. If don't understand ask us, no problem.

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt:17]: Why do you need this? 🤔

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: Same as above

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt]: You should do this in an if/else statement. `if (list.isEmpty) showEmptyNews else showNews`

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: You should rename this presenter function to onInit and make private the setListNews function, so, call setListnews function from onInit. Remember, the fragment allways ask what should make?

**Review by ffasitella — CHANGES_REQUESTED** [review]
Left some comments. Check this PR, it has some merge conflicts. You shoul merge master to this branch to solve them.
Also, add a screenshot to show us how this looks 👀

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt]: Rename this function to any more descriptive, the presenter don't know about Recycler Views. You could rename to loadNewsList.

File: app/src/main/res/drawable/ic_add.xml
Comment (jcruz-wolox) [inline app/src/main/res/drawable/ic_add.xml]: Move to dimens.xml file the width and height values.

File: app/src/main/res/drawable/ic_favorite.xml
Comment (jcruz-wolox) [inline app/src/main/res/drawable/ic_favorite.xml]: Same as above, move to dimens.xml width and height values.

File: app/src/main/res/drawable/ic_favorite_border.xml
Comment (jcruz-wolox) [inline app/src/main/res/drawable/ic_favorite_border.xml]: Same as above.

File: app/src/main/res/drawable/ic_news_icons.xml
Comment (jcruz-wolox) [inline app/src/main/res/drawable/ic_news_icons.xml]: Same as above.

File: app/src/main/res/layout/fragment_news.xml
Comment (jcruz-wolox) [inline app/src/main/res/layout/fragment_news.xml]: Remove empty lines.

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsPresenter.kt]: You could rename to onRefreshNews and call setListNews() function

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (jcruz-wolox) [inline app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: Call to onRefreshNews() function. Below I explain how

**Review by jcruz-wolox — CHANGES_REQUESTED** [review]
Hello Félix!! 

I have any comments.

One suggestion, you can make comments from here (Github PR), i think that you is making comments by responding the github notifications emails, shouldn't response them because it generates spam in the PR.

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt
Comment (FloresFelix) [reply app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: sorry I don't understand...

Comment (jcruz-wolox) [reply app/src/main/java/ar/com/wolox/android/example/ui/viewpager/news/NewsFragment.kt]: Sorry, I confused you. The idea is that you convert to private setListNews and create onInit function to call it here. Example:

NewsFragment.kt

`override fun init() {
    presenter.onInit()
}`

-------------------------------

Presenter:

```
// New function called onInit()
fun onInit() {
    setListNews()
}
```

```
// the setListNews function converted to private. you'll be able to call this function also when you refresh the news list
private fun setListNews() {
 
}
```

**Review by ffasitella — APPROVED** [review]
Good job! In a real project, you will fight merge conflicts very often. Well done 👍🏽 
Let's wait for Jona's approval!

**Review by jcruz-wolox — APPROVED** [review]

**Pull request merged**
